### PR TITLE
Fixed broken link to "Dive into Python"

### PR DESCRIPTION
**Pull request by mandarvaze**

### CODE REVIEW

File: docs/narr/testing.rst [hunk 1]
```diff
@@ -50,7 +50,7 @@ The suggested mechanism for unit and integration testing of a :app:`Pyramid`
 application is the Python :mod:`unittest` module.  Although this module is
 named :mod:`unittest`, it is actually capable of driving both unit and
 integration tests.  A good :mod:`unittest` tutorial is available within `Dive
-Into Python <http://www.diveintopython.net/unit_testing/index.html>`_ by Mark
+Into Python <https://www.diveinto.org/python3/unit-testing.html>`_ by Mark
```
Comment (stevepiercy) [inline docs/narr/testing.rst]: This is now "Dive into Python 3". Would you please make this additional correction? Thank you!

Comment (mandarvaze) [reply docs/narr/testing.rst]: will do.

**Comment by stevepiercy** [comment]
The failure is unrelated to this PR as it appears that docs.pytest.org is temporarily down. Merging.

**Comment by stevepiercy** [comment]
Thank you!

**Pull request merged**